client/packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -116,6 +117,13 @@ export const HeaderCell = <T extends RecordWithId>({
     child
   );
 
+  const tooltipPlacement =
+    align === 'left'
```

Comment:
Ooh.. I kinda don't like this nested tenary...

Reply:
if `switch` would be an expression in JS (like in Rust)...

but you are too fast already changed it with some other refactors when fixing the table cells as well ;)